packages/core/src/baseclient.ts

```diff
@@ -237,6 +238,22 @@ export abstract class BaseClient<O extends ClientOptions> implements Client<O> {
     }
   }
 
+  /**
+   * Sends user feedback to Sentry.
+   */
+  public captureUserFeedback(feedback: UserFeedback, _hint: EventHint = {}): void {
+    const envelope = createUserFeedbackEnvelope(
+      feedback,
+      this.getDsn(),
+      this.getSdkMetadata(),
+      this.getOptions().tunnel,
+    );
+
+    // sendEnvelope should not throw
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises
+    this.sendEnvelope(envelope);
+  }
```

Comment:
In the next step this is going to incorporate sending more fields, be merging https://github.com/getsentry/sentry-javascript/blob/311638cd98712ad95c62d6b8d654199d7f84f36f/packages/feedback/src/core/sendFeedback.ts#L11 into here and handling both sets of args.

So primarily `attachments`, and setting a `source` will be new, which I think can both come in through the hint object.